PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -255,12 +255,16 @@ static bool MapKeyChar(string input, ref ConsoleModifiers mods, out ConsoleKey k
                             keyChar = (char)(keyChar - 'A' + 1);
                             break;
                         case '_':
-                            keyChar = Keys.CtrlUnderbar.KeyChar;
-                            mods |= Keys.CtrlUnderbar.Modifiers;
+                            keyChar = '\x1f';
```

Comment:
I don't know the history why <kbd>Ctrl+_</kbd> gets mapped to `0x1f`, but I'm reasonably sure it's been that way since before I was born.

I could introduce a constant to be more readable, but not in this PR.